pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React from "react";
+
+import { CardBody } from "@patternfly/react-core/dist/esm/components/Card/index.js";
+import { DescriptionList } from "@patternfly/react-core/dist/esm/components/DescriptionList/index.js";
+
+import { StorageCard, StorageDescription, new_card, new_page } from "../pages.jsx";
+import { StorageUsageBar } from "../storage-controls.jsx";
+import { get_fstab_config_with_client } from "../utils.js";
+import { btrfs_usage } from "./utils.jsx";
+import { mounting_dialog } from "../filesystem/mounting-dialog.jsx";
+import { check_mismounted_fsys, MismountAlert } from "../filesystem/mismounting.jsx";
+import { is_mounted, mount_point_text, MountPoint } from "../filesystem/utils.jsx";
+import client from "../client.js";
+
+const _ = cockpit.gettext;
+
+function subvolume_unmount(volume, subvol, mount_point) {
+    const block = client.blocks[volume.path];
+    // HACK: generalise this? Instead of a global findmnt state, put the active mount point in subvol.
+    subvol.mount_point = mount_point;
```

Comment:
Is this actually used anywhere?  And isn't `mount_point` the mount point from fstab?

Reply:
Uh, this was a hack, and is now replaced by `client.btrfs_mounts`, thanks for spotting this!